packages/spacecat-shared-content-client/src/clients/content-client.js

```diff
@@ -20,9 +20,15 @@ import { GetSecretValueCommand, SecretsManagerClient } from '@aws-sdk/client-sec
 const CONTENT_SOURCE_TYPE_DRIVE_GOOGLE = 'drive.google';
 const CONTENT_SOURCE_TYPE_ONEDRIVE = 'onedrive';
 
+/**
+ * @import {type Site} from "@adobe/spacecat-shared-data-access/src/models/site/index.js"
```

Comment:
I'll send a follow-up for the type definitions in spacecat-shared-data-access later